docs/detections/prebuilt-rules-management.asciidoc

```diff
@@ -6,7 +6,7 @@
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [manage]
 
-Follow these guidelines to start using the {es-sec-app}'s <<prebuilt-rules, prebuilt rules>>, keep them updated, and make sure they have the data needed to run successfully. 
+Follow these guidelines to start using the {security-app}'s <<prebuilt-rules, prebuilt rules>>, keep them updated, and make sure they have the data needed to run successfully. 
```

Comment:
For my own edification, when should we use {security-app} over {elastic-sec}? I feel like {elastic-sec} would work here too, and might even be more appropriate to use since the security rules are actually from Elastic.

Reply:
@nastasha-solomon Good question, I added to the team sync agenda, since it should probably be a separate discussion with the whole team. This PR is just removing the dependency on book-scoped attributes. But I think at one point, Elastic might have wanted to call the solutions "apps", though I'm not sure that's still the preferred term.